.github/workflows/publish.yml

```diff
@@ -65,9 +70,12 @@ jobs:
         run: ./bash/docker_container_setup.sh
 
       - name: publish
+        env:
+          latest: ${{ github.event.inputs.latest == 'true' && '--latest' || '' }}
```

Comment:
interestingly this only works with `github.event.inputs.latest` and not `inputs.latest`. Not sure if the latter can truly be treated as a boolean, in general in GHA boolean logic is a little funny